src/utils.ts

```diff
@@ -289,7 +288,7 @@ export const WalletSelectorTransactions = (
     };
   });
 
-  return { transactions: parsedTransactions } as SignAndSendTransactionsParams;
+  return { transactions: parsedTransactions };
 };
 
```

Comment:
Are we sure about this one?

Reply:
`Module '"@near-wallet-selector/core/lib/wallet"' has no exported member 'SignAndSendTransactionsParams'.ts(2305)`

It was deleted and I can't run `npm install` it with this line:
https://github.com/near/wallet-selector/commit/65ea358c96859922e8cc694ac0b9e76f4ddf60de#diff-cf1e42785f014c407e1c8f9672f2d13774005ebee6c4b8b65927b56e0f72ecb0L46